app/assets/javascripts/admin/modules/unpublish-display-conditions.js

```diff
@@ -17,6 +17,25 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   UnpublishDisplayConditions.prototype.init = function () {
     this.initUnpublishTypeListener()
+    this.initUnpublishRedirectListener()
+  }
+
+  UnpublishDisplayConditions.prototype.initUnpublishRedirectListener = function () {
+    var checkbox = this.module.querySelector('.govuk-checkboxes__item input[name="unpublishing[redirect]"]')
+
+    checkbox.addEventListener('change', function (e) {
+      this.showPublicExplanation(!e.target.checked)
```

Comment:
```suggestion
      this.showPublicExplanation( e.currentTarget.checked )
```

We should use `currentTarget` instead of `target` as `target` could be another element, see this [answer](https://stackoverflow.com/a/10086501).

Also it's better to pass in the `checked` value rather than the negative as it makes the code more readable.